subprojects/precondition-tester/src/test/groovy/org/gradle/test/precondition/PreconditionProbingTest.groovy

```diff
@@ -32,7 +31,7 @@ import org.junit.jupiter.api.Assumptions
  *         This is normal, and simply signifies that the precondition is not satisfied on the current system
  *     </li>
  *     <li>
- *         <b>Failed:</b> there was an error meanwhile checking the combination.
+ *         <b>Failed:</b> there was an error while checking the combination.
```

Comment:
💭 Why are there so many different message prefixes when reporting an error:

- Problem while ...
- An error happened while ...
- Failed: there was an error while ...

Could these be standardized?

Reply:
Good point, my pet peeve (as I just learned not "beef" :D ) is not yet big enough with this.